beacon-chain/sync/pending_attestations_queue_test.go

```diff
@@ -259,54 +252,72 @@ func TestProcessPendingAtts_HasBlockSaveUnAggregatedAttElectra(t *testing.T) {
 }
 
 func TestProcessPendingAtts_NoBroadcastWithBadSignature(t *testing.T) {
```

Comment:
This test was badly broken. The "wrong signature" attestations failed validation because of a different reason, not because of the signature. There was also a lot of unnecessary duplication such as creating two services.